.github/workflows/ci.yml

```diff
@@ -399,6 +399,7 @@ jobs:
       - name: miri
         run: |
           cargo miri nextest run --features full --lib --tests --no-fail-fast
+          cargo miri test --doc --all-features --no-fail-fast
```

Comment:
The miri job is starting to take a rather long time. Could you add a new job for this, instead of adding additional work to the existing job? This will allow for parallelism.